assets/src/scripts/components/MoreInfoModal.tsx

```diff
@@ -176,6 +178,22 @@ function MoreInfoModal({
               )}
             </ul>
           )}
+          <h2 className="h6 font-weight-bold">References</h2>
+          {loading ? (
+            <p>Loading references...</p>
+          ) : (
+            <ul>
+              {!references || references?.length === 0 ? (
+                <li>No references</li>
+              ) : (
+                references.map((reference, idx) => (
+                  <li key={idx}>
+                    <a href={reference[1]}>{reference[0]}</a>
+                  </li>
+                ))
+              )}
```

Comment:
We can update this and the synonyms above to use optional chaining to check the array length, and simplify this comparison:

```suggestion
              {!references?.length ? (
                <li>No references</li>
              ) : (
                references.map((reference, idx) => (
                  <li key={idx}>
                    <a href={reference[1]}>{reference[0]}</a>
                  </li>
                ))
              )}
```